src/LayoutManager/README.md

```diff
@@ -2,6 +2,39 @@
 
 Layout manager exposes a main public method that performs group layout given a target group and target objects.
 
+## Expectations for non interactive groups
+
+The layout manager will calculate the size and position of the group when is created or updated
+The FitContent strategy is the classic fabric 5.x group and will grow or shrink when objects are added or removed.
+In general the expectation is that creating or modifying an object will create a group with a postion and once that group is added to canvas, the position of the objects will be the same visually that those object would have had on the canvas if they weren't on the group.
+This is a legacy behaviour that carries over this new code.
+The behaviour is not important if you are building a group from scratch and you are adding it in a specific position on the canvas, in that case the original position is meaningless and you are overriding it by specifying your postion.
+
+Since inside the group all objects are organized around the center is anyway easier if you can think of your objects position and alignment with scene coordinates and know that they will be maintained in the same position when grouped.
```

Comment:
```suggestion
Since inside the group all objects are organized around the center is anyway easier if you can think of your objects position and alignment with scene coordinates and know that they will be maintained in the same position when grouped only if you did not pass transform options.
```

Reply:
This is very misleading